pdk/1.0/modules/ROOT/pages/policies-pdk-troubleshooting.adoc

```diff
@@ -4,54 +4,115 @@ include::_attributes.adoc[]
+Anypoint CLI requires Multi-factor authentication (MFA) with a Connected App. Using any PDK command with out Connected App credentials throws an error similar to the following:
 
-Caused by:
-  authenticated registries require a credential-provider to be configured
-  see https://doc.rust-lang.org/cargo/reference/registry-authentication.html for details
-make: *** [install-cargo-anypoint] Error 101
+[source,ssh]
 ----
+Error: Failed to launch the browser process! undefined
+[...]:ERROR:ozone_platform_x11.cc(239)] Missing X server or $DISPLAY
+[21921:21921:1103/113335.217708:ERROR:env.cc(255)] The platform failed to initialize. Exiting.
+TROUBLESHOOTING: https://pptr.dev/troubleshooting
+----
+
+To configure Anypoint CLI with MFA authentication using a Connected App, refer to xref:4.x@anypoint-cli::auth.adoc[].
 
-To fix the error:
+[[group-id-error]]
+== Group ID Error When Creating the Policy Project
 
-. Find your Cargo main directory (`.cargo/`).
-. In `.cargo/`, find the `credentials.toml` file and add the following new line `credential-provider = ["cargo:token"]` to the '[registries.anypoint]` section, for example:
+Anypoint CLI uses the `ANYPOINT_ORG` environment variable. This variable can use either the Organization Name or the Organization ID as the value. If the variable is set to the Organization Name, the xref:policies-pdk-create-project.adoc[policy creation command] fails to infer the Group ID of the organization. 
+
+It is recommended to use the Organization ID for the value `ANYPOINT_ORG`. However, if you cannot change the variable, there are two options to provide the Group ID:
+
+* If you run the create command and receive the following prompt:
 +
-[source,toml]
+[source,ssh]
 ----
-[registries.anypoint]
-token = <someToken>
-credential-provider = ["cargo:token"]
+Please provide a valid group-id (the id of the organization that will own the asset):
 ----
 +
-NOTE: Do not edit the `token` field.
+Enter your Group ID, then press enter.
+
+* Or, use the `--group-id` flag when creating the PDK project, for example:
++
+[source,ssh]
+----
+anypoint-cli-v4 pdk policy-project create -n <policy-name> --group-id <organization-id>
+----
+
+[[authorization-error]]
+== 401 Authorization Error When Running Make Setup
+
+When executing `make setup', the following error might occur:
+
+[source,ssh]
+----
+$ make setup
+       Login token for `anypoint` saved
+[...]
+
+Caused by:
+  failed to get successful HTTP response from `https://anypoint.mulesoft.com/crates[...]/download` [...], got 401
+  body:
+make: *** [install-cargo-anypoint] Error 101
+----
+
+This error occurs when your version of Rust is not {rust-ver-var} or later.
 
-. Save the changes.
-. Run the `setup` or `build` command from your custom policy root directory again. The command should now execute properly.
+To solve this error, refer to xref:policies-pdk-prerequisites.adoc#rust-requirements[Rust Requirements] to update your Rust requirements.
```

Comment:
Update your requirements? Or "ensure that your installation meets the requirements"?